packages/bootstrap-4/src/ArrayFieldTemplate/ArrayFieldTemplate.tsx

```diff
@@ -24,12 +25,23 @@ const ArrayFieldTemplate = (props: ArrayFieldTemplateProps) => {
     schema,
     title,
   } = props;
-  const {
-    ArrayFieldDescriptionTemplate,
-    ArrayFieldItemTemplate,
-    ArrayFieldTitleTemplate,
-  } = registry.templates;
   const uiOptions = getUiOptions(uiSchema);
+  const ArrayFieldDescriptionTemplate =
+    getTemplate<"ArrayFieldDescriptionTemplate">(
```

Comment:
Do we need the generic argument in these cases? Seems like in some cases it's inferred from the first arg, in other places like this it's explicit.

Reply:
The places where it is inferred are JS files which do not support generics. The <Name> generic causes the appropriately casted return value to be exactly the right function. Super helpful... now if only I could have figured out how to use the generic as a variable to access the array